lib/include/hse_ikvdb/kvset_view.h

```diff
@@ -12,9 +12,9 @@ struct cn;
 struct kvset;
 
 struct kvset_metrics {
-    uint32_t num_keys;
-    uint32_t num_tombstones;
-    uint32_t nptombs;
+    uint64_t num_keys;
```

Comment:
Good catch. Hope the users of these fields are using the right type as well..

Reply:
Thx!  I noticed this in the metrics "totals" output when the key count went over 4-billion :-)